src/auro-menu.js

```diff
@@ -100,11 +100,6 @@ class AuroMenu extends LitElement {
   }
 
   updated(changedProperties) {
-    this.addEventListener('slotchange', () => {
-      // make sure to update all menuoption noCheckmark attributes when the menu is dynamically changed
-      this.handleNoCheckmarkAttr();
-    });
-
```

Comment:
I get your point for this one.
I think I have two reason, first I move this part because I think it is redundant with the `handleSlotItems` handler (so I move it there).
Second, isn't this going to keep adding the event listener to the slot element? not sure for this one though.

If I may, I might still keep this changes, but also will change something in the `handleSlotItems` handler, in which I will need advice for the changes in the other comment.
Thanks.